### DISCUSSION TITLE
灌水

### DISCUSSION THREAD

**Discussion by hutao48** [share121/inter-knot#5478] [灌水]
和标题一样，为了灌水而灌水
![20240810_092626](https://github.com/user-attachments/assets/107c7648-05fe-4937-a179-098ceb13bf78)

**Comment by Ayn1631** [comment]
哪个画师啊

**Reply by hutao48** [reply]
有没有一种可能，我是照着网图自己画的

**Reply by bigetree** [reply]
好看【鼓掌】

**Reply by Ayn1631** [reply]
> hutao48

太强了!

**Comment by March-777th** [comment]
## 我就说绳网有大佬吧<img src="https://img-static.mihoyo.com/communityweb/upload/a370edf80a844b3abcececf055ab9cf9.png" alt="三月七_盯" width="75" >
~我是fw~

**Reply by KELERSAMA** [reply]
大佬

**Comment by XiaoYumuamua** [comment]
豪勘
